solutions/remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -402,6 +447,219 @@
OM\r\n        TelemetryAndRules TA PARTITION BY __partitionid\r\n    WHERE\r\n        TA.AggregationWindow='tumblingwindow5minutes'\r\n    GROUP BY\r\n        TA.__deviceid,\r\n        TA.__ruleid, \r\n        TA.__partitionid,\r\n        TA.MeasurementName,\r\n        TumblingWindow(minute, 5)\r\n),\r\nGroupAggregatedMeasurements AS (\r\n    SELECT\r\n        WA.__deviceid,\r\n        WA.__ruleid,\r\n        WA.__partitionid,\r\n        WA.__lastReceivedTime,\r\n        Collect() AS Measurements\r\n    FROM\r\n        WindowAggregates5Min WA PARTITION BY __partitionid\r\n    GROUP BY \r\n        WA.__deviceid,\r\n        WA.__ruleid,\r\n        WA.__partitionid,\r\n        WA.__lastReceivedTime,\r\n        System.Timestamp\r\n),\r\nFlatAggregatedMeasurements AS (\r\n    SELECT\r\n        GA.__deviceid,\r\n        GA.__ruleid,\r\n        GA.__lastReceivedTime,\r\n        udf.flattenMeasurements(GA) AS __aggregates\r\n    FROM\r\n        GroupAggregatedMeasurements GA PARTITION BY __partitionid\r\n),\r\nCombineRuleFilters AS (\r\n    SELECT\r\n        FA.__deviceid,\r\n        FA.__ruleid,\r\n        FA.__aggregates,\r\n        FA.__lastReceivedTime,\r\n        R.Description as __description,\r\n        R.Severity as __severity,\r\n        R.__rulefilterjs as __rulefilterjs\r\n    FROM\r\n        FlatAggregatedMeasurements FA PARTITION BY __partitionid\r\n        JOIN Rules R ON FA.__ruleid=R.Id\r\n),\r\nApplyRuleFilters AS\r\n(\r\n    SELECT\r\n        CR.*\r\n    FROM\r\n        CombineRuleFilters CR PARTITION BY __partitionid\r\n    WHERE TRY_CAST(udf.applyRuleFilter(CR) AS bigint) = 1\r\n)\r\n\r\nSELECT\r\n    1 as [doc.schemaVersion], \r\n    'alarm' as [doc.schema], \r\n    'open' as [status], \r\n    '1Rule-1Device-NMessage' as [logic], \r\n    DATEDIFF(millisecond, '1970-01-01T00:00:00Z', System.Timestamp) as created, \r\n    DATEDIFF(millisecond, '1970-01-01T00:00:00Z', System.Timestamp) as modified, \r\n    AR.__description as [rule.description], \r\n    AR.__severity as [rule.severity], \r\n    AR.__ruleid as [rule.id], \r\n    AR.__deviceId as [device.id],\r\n    AR.__aggregates,\r\n    AR.__lastReceivedTime as [device.msg.received]\r\nINTO\r\n    Alarms\r\nFROM\r\n    ApplyRuleFilters AR PARTITION BY __partitionid\r\n\r\nSELECT\r\n        CONCAT(T.IoTHub.ConnectionDeviceId, ';', CAST(DATEDIFF(millisecond, '1970-01-01T00:00:00Z', T.EventEnqueuedUtcTime) AS nvarchar(max))) as id,\r\n        1 as [doc.schemaVersion],\r\n        'd2cmessage' as [doc.schema],\r\n        T.IoTHub.ConnectionDeviceId as [device.id],\r\n        'device-sensors;v1' as [device.msg.schema],\r\n        DATEDIFF(millisecond, '1970-01-01T00:00:00Z', System.Timestamp) as [device.msg.created],\r\n        DATEDIFF(millisecond, '1970-01-01T00:00:00Z', T.EventEnqueuedUtcTime) as [device.msg.received],\r\n        udf.removeUnusedProperties(T) as Data\r\nINTO\r\n    Messages\r\nFROM\r\n    DeviceTelemetry T PARTITION BY PartitionId TIMESTAMP BY T.EventEnqueuedUtcTime\r\n",
```

Comment:
We should make this parameters with default value so that it can be changed through deployment if needed